src/QAT/qat/purr/backends/live.py

```diff
@@ -30,6 +31,76 @@
 log = get_default_logger()
 
 
+def apply_setup_to_hardware(hw: QuantumHardwareModel,
```

Comment:
`apply_lucy_hardware(...)` and hardcode all values to lucy, you should only have the hardware as an argument.

Reply:
Agree